core/templates/services/html-escaper.service.spec.ts

```diff
@@ -19,14 +19,21 @@
 import { TestBed } from '@angular/core/testing';
 
 import { HtmlEscaperService } from 'services/html-escaper.service';
+import { importAllAngularServices } from 'tests/unit-test-utils';
 
```

Comment:
`core/templates/services/html-escaper.service.spec.ts` was made strictly typed before merge of this PR. This import is making strict checks to fail. Is there any alternative for this ?